.circleci/config.yml

```diff
@@ -87,6 +87,7 @@ jobs:
           name: 📷 Run Percy snapshots
           command: |
             cd docs
+            sudo apt install libgconf-2-4 libatk1.0-0 libatk-bridge2.0-0 libgdk-pixbuf2.0-0 libgtk-3-0 libgbm-dev libnss3-dev libxss-dev
```

Comment:
What is all this ?

Reply:
> What is all this ?

Without those i was getting below error -
<img width="1280" alt="Screen Shot 2022-04-14 at 9 24 42 PM" src="https://user-images.githubusercontent.com/21966124/163422474-487557eb-4d3c-4f16-be2c-ff4769d206f8.png">